frontend/src/component/pages/admin/changelog/ManageChangeLogs.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { Section } from '../../../ui/Section';
+import { Button } from '../../../ui/Button';
+import { Store } from 'redux';
+import { IAppState } from '../../../../state/reducers';
+import { ChangeLogService } from '../../../../service/ChangeLog.service';
+import { Modal } from '../../../ui/Modal';
+import './ManageChangeLogs.scss';
+import { CreateChangeSection } from './CreateChangeSection';
+import { Icon, IconID } from '../../../ui/Icon';
+
+interface IProps {
+  changeLogService: ChangeLogService;
+  notifyToast: (message: string, duration?: number) => void;
+  store: Store<IAppState>;
+  onAuthenticationFailed: () => void;
+}
+
+export class ManageChangeLogs extends Component<IProps> {
+  private createChangeModalRef = React.createRef<Modal>();
+
+  render() {
+    return (
+      <div className="manage-change-logs">
+        <Section
+          title={'Manage Change Logs'}
+          options={[this.renderCreateButton()]}
+        />
+        <Modal ref={this.createChangeModalRef} canClose={true}>
+          <div className={'create-change-modal-close-icon'}>
+            <Icon
+              defaultIconID={IconID.Close}
+              onClick={this.handleOnCreateChangeModalCloseClick}
+            />
+          </div>
+          <div className={'create-change-section-container'}>
+            <CreateChangeSection
+              onChangeCreated={this.handleOnChangeCreated}
+              changeLogService={this.props.changeLogService}
+              onAuthenticationFailed={this.props.onAuthenticationFailed}
+              store={this.props.store}
+            />
+          </div>
+        </Modal>
```

Comment:
I recommend we consider wrapping this into `CreateChangeModal` component so that the readers are not distracted by unrelated implementation details when reading this class.